Azure/communication-ui-library-android
# [Calling][Feature] Skip Setup Screen feature

## Purpose

* Includes skip setup screen capability
* Includes camera & mic on/off API

## Does this introduce a breaking change?



## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull Request Checklist




## How to Test


* Open demo app 
* Check skip setup screen checkbox in settings
* Launch call


## What to Check
Verify that the following are valid
* To see if it can bypass the setup screen

## Other Information
